### ISSUE TITLE
[GSSoC'21] adding dark mode with toggle button

### ISSUE THREAD

**Issue by arpit456jain** [Spectrum-CETB/LesKollab#75]
I want to add a dark mode to the website as i had already raised this issue before and made a PR but by mistake i closed the PR and issue was also closed i had already added this feature to website just had to make a PR

**Comment by arpit456jain** [comment]
@rupeshmohanty kindly assign me this issue again sorry for inconvenience seriously I don't know how i closed that PR its my work of 3 hours how can i closed it i can't believe my self.

**Comment by rupeshmohanty** [comment]
@arpit456jain Did that PR get accepted?

**Comment by arpit456jain** [comment]
<img width="960" alt="les" src="https://user-images.githubusercontent.com/55352601/114189537-ece45400-9967-11eb-868c-e5cabceb6dbf.png">

This was the PR you added the labels and request the changes.
This morning when I saw you closed the issue then I notice may be I had closed the PR by mistake

**Comment by rupeshmohanty** [comment]
There was already a PR for this issue, so that's why I closed this issue.

**Comment by arpit456jain** [comment]
no sir I think some misunderstanding happens.
1 .I had raised a issue for adding dark mode which was assign to me.
2 I had created a PR for that.
3 then last night by mistake I had closed the PR.
4 This morning you closed my previous issue for that may be because I had closed the PR.
5. Then I created this new issue for dark mode.

If some else is working on this then I have no problem. 
Otherwise kindly assign me this issue I already done the work yesterday just have to make a PR.
